### DISCUSSION TITLE
Trying out GitHub Discussions

### DISCUSSION THREAD

**Discussion by kfix** [kfix/ddcctl#121] [Announcements]
## 👋 Welcome!
  I'm opening up the GitHub Discussions for the `ddcctl` project to provide a place to connect with other users.

I hope this is a better way to troubleshoot the usual "integration issues" when using DDC from macOS than filing not-bugs on the GitHub Issues tracker.

Haven't decided on categories for structuring the forum yet, but posts on the following topics are appropriate:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Troubleshooting monitor + dock/adapter/cable + OS combinations.

**Comment by DenisVASI9** [comment]
Hi, Joey.
I'm sorry to write here off-topic, I couldn't find any other way to write to you.
Did you boot grub through kexec, I would like to ask a couple of questions, how can I do ?

**Reply by kfix** [reply]
that is [ancient history](https://lists.gnu.org/archive/html/grub-devel/2009-04/msg00367.html)!

No, I did not have any success running grub2 via kexec. only grub4dos supported it, but it crashed much of the time when started that way.

That was also on x86-32 BIOS hardware. 

the "weird reasons" were that I was remotely converting servers from FreeBSD<->Linux without having a DHCP network (so, no PXE boot) and no IPMI/DRAC/ILO) to remotely-mount an .iso file. just a serial-port console. GRUB2 could be compiled on and boot from both platforms, but rebooting to real mode and ensuring core.img was always in just the right spot of the boot drive's unpartitioned front and getting the menu file correct for autoboot with serial console & manual IP addressing with custom liveboot ramdisks, etc. etc. what a mess! kexec seemed like a way to at least skip the risk of ruining the working bootloader.

**Comment by joeldcanfield** [comment]
thanks for a no-nonsense tool. finally hooked my Ubuntu box to my Mac's dual monitor system, and about 4 minutes ago Googled something about controlling monitor inputs from Terminal, and it's already working and ready for scripting and keyboard shortcuts in Hammerspoon.

**Comment by vineetsrlkr** [comment]
how does this really work ? does it use the brightness slider from control centre?
